stores/wallet.ts

```diff
@@ -38,6 +39,7 @@ export const useWalletStore = defineStore('wallet', {
     disconnecting: false,
   }),
   getters: {
+    getWalletVM: state => state.selected?.vm || 'SUB',
```

Comment:
actually this is not accurate , you might be logged out , and on a evm drop and this would return 'SUB'

```suggestion
    getWalletVM: state => state.selected?.vm,
```

![CleanShot 2024-08-09 at 10 50 08@2x](https://github.com/user-attachments/assets/7168eb25-40ce-4af1-b56e-71386dcdfa36)